docs/content/customization/generic/global-mirror.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Global Image Registry Mirror"
++++
+
+Add containerd image registry mirror configuration to all Nodes in the cluster.
+
+When the `globalImageRegistryMirror` variable is set, `files` with configurations for [Containerd default mirror](https://github.com/containerd/containerd/blob/main/docs/hosts.md#setup-default-mirror-for-all-registries) will be added.
+
+This customization will be available when the
+[provider-specific cluster configuration patch]({{< ref "..">}}) is included in the `ClusterClass`.
+
+## Example
+
+To provide image registry mirror with CA certificate, specify the following configuration:
+
+If your registry mirror requires a private or self-signed CA certificate,
```

Comment:
The API supports credential with username/password as well. Do we want to document an example for username/password, too?

Reply:
The API supports credentials for username and password for mirror, but it is not implemented for the globalImageRegistryMirror yet. user can provide mirror credentials in the `imageRegistries` block for now. 
Once we implement the support for mirror credentials explicitly, we can update this documentation.

Reply:
Thanks, I missed that.

If I create a Secret with username/password and reference it, will I get an error? I would want that.

Reply:
Added https://github.com/d2iq-labs/capi-runtime-extensions/issues/329 to track the enhancement.

Reply:
So, right now, a ref handles a Secret with a certificate, but not the username/password. That's fine. Thanks!